pkg/provider/k8s/k8s.go

```diff
@@ -546,6 +549,44 @@ func (c *K8s) jobApply(resource runtime.Object) error {
 		func() (bool, error) { return c.jobReady(resource) })
 }
 
+func (c *K8s) validatingWebhookConfigurationApply(resource runtime.Object) error {
+	req := resource.(*admissionregistration.ValidatingWebhookConfiguration)
+	kind := resource.GetObjectKind().GroupVersionKind().Kind
+
+	switch v := resource.GetObjectKind().GroupVersionKind().Version; v {
+	case "v1":
+		client := c.clt.AdmissionregistrationV1().ValidatingWebhookConfigurations()
+		list, err := client.List(c.ctx, apiMetaV1.ListOptions{})
+		if err != nil {
+			return errors.Wrapf(err, "error listing resource : %v, name: %v", kind, req.Name)
+		}
+		var exists bool
+		for _, l := range list.Items {
+			if l.Name == req.Name {
+				exists = true
```

Comment:
think adding the following code can resolve the issue

	if req.ResourceVersion == "" {
		req.ResourceVersion = l.ResourceVersion
	}